packages/components/src/components/text-field/text-field.tsx

```diff
@@ -172,6 +184,14 @@ export class TextField {
             {...(this.helperText ? ariaDescribedByAttr : {})}
           />
 
+          { this.list && (<datalist id="data-list">
```

Comment:
I would remove this and let the user define it themselves. Mainly to keep things simple, and also because ids need to be unique (in valid HTML), so there cannot be more than one "data-list" id in the document.

So people using the library would do it like this:

```html
<scale-text-field label="List of things" list="things"></scale-text-field>
<datalist id="things">
  <!-- options here -->
</datalist>
```

Reply:
Hi @acstll,
Thanks for your suggestions. I did not get enough time to look into this. I will work on the review changes in a while.

Thanks again!